.ci/packaging.groovy

```diff
@@ -79,6 +78,10 @@ pipeline {
                 name 'PLATFORM'
                 values 'linux && immutable', 'arm'
               }
+              axis {
+                name 'TYPE'
+                values 'snapshot', 'staging'
```

Comment:
is it possible to also exclude staging here when on main?

Reply:
I tried to use https://github.com/elastic/apm-server/pull/7584/commits/eaa81b1b0ddaeb10462e7832c5f186bd78f11ac4 (the declarative form) but somehow the UI representation skipped one of the stages.

<img width="874" alt="image" src="https://user-images.githubusercontent.com/2871786/160570918-95d7c201-6fd6-4536-832f-df7c0ed20717.png">

So I finally decided to use the scripted approach with https://github.com/elastic/apm-server/pull/7584/commits/e46eb9bac19a6baffdb2feeff223be59062a1b83

<img width="1462" alt="image" src="https://user-images.githubusercontent.com/2871786/160571175-d426110e-45e5-46f1-ac0c-8d57118a76ca.png">